dao/src/integ-test/java/uk/gov/moj/sdt/dao/IndividualRequestDaoTest.java

```diff
@@ -146,7 +151,7 @@ public void testGetBulkSubmissionLower() {
 
         createIndividualRequest(customerRequestReference, LocalDateTime.now().plusDays(dataRetentionPeriod * -1));
         final IIndividualRequest individualRequest =
-                individualRequestDao.getIndividualRequest(bulkCustomer, customerRequestReference, dataRetentionPeriod);
+                individualRequestDao.getIndividualRequest(bulkCustomer, customerRequestReference, dataRetentionPeriod + 1);
```

Comment:
Again, not sure why adding 1 to data retention period.

Reply:
removed this now